### DISCUSSION TITLE
Do we need early exit multiples on SAFEConversionMechanism?

### DISCUSSION THREAD

**Discussion by pjohnmeyer** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#265] [Suggestions]
We have them on [`NoteConversionMechanism`](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/v1.0.0-b1/docs/schema/types/conversion_mechanisms/NoteConversionMechanism.md) but not on [`SAFEConversionMechanism`](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/v1.0.0-b1/docs/schema/types/conversion_mechanisms/SAFEConversionMechanism.md.) Our system allows this to be optionally recorded.

**Comment by JSv4** [comment]
I'd vote yes. I'll create an issue.
